### ISSUE TITLE
This matconvnet version may not use fixed moments in batch normalization when call net.eval

### ISSUE THREAD

**Issue by chinakook** [peiyunh/tiny#27]
As usual, we want to use fixed moments when data flow into the batch normalization layer in a testing case, this may help us to improve the test performance. However, When I made a calculation by hand with the net.vars in the matlab command window, I had got that the net.eval HAD NOT USE the fixed moments in the net.params but the moments calculated according to the input image(even while the net.mode was set to 'test'). That may reduce the performance in the test case.

**Comment by peiyunh** [comment]
You can set `net.mode='test'` to force the network to use fixed moments. See [here](https://github.com/peiyunh/matconvnet/blob/d972fe848900c1c56d32e7a955ca38d8e05e0669/matlab/%2Bdagnn/BatchNorm.m#L14-L24)

**Comment by chinakook** [comment]
I got it. I miss a net.mode='test' in my own script but the origin tiny_face_detector.m contain it.
